angular2-jwt.ts

```diff
@@ -1,19 +1,14 @@
-import { Injectable } from '@angular/core';
+import { Injectable, Provider } from '@angular/core';
 import { Http, Headers, Request, RequestOptions, RequestOptionsArgs, RequestMethod, Response } from '@angular/http';
 import { Observable } from 'rxjs/Observable';
-import 'rxjs/add/observable/fromPromise';
```

Comment:
You need the mergemap and fromPromise operators added here as otherwise it's a potential source of bugs for users who aren't already including them (unless things changed after beta 6. I haven't been keeping track).